recipes/abyss/meta.yaml

```diff
@@ -16,18 +16,19 @@ source:
 
 requirements:
   build:
-  - {{ compiler('c') }}
-  - boost
-  - sparsehash
-  - sqlite
-  - openmpi
+    - {{ compiler('cxx') }}
+  host:
+    - boost
+    - sparsehash
+    - sqlite
+    - openmpi
```

Comment:
@bgruening Are these dependencies listed in `host` required to be installed on the end-user's computer that runs ABySS, or only on the machine that builds the package for ABySS? I read https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#host but wasn't quite clear on the answer to my question afterward.